app/src/main/java/com/nextcloud/talk/ui/dialog/MessageActionsDialog.kt

```diff
@@ -64,16 +68,24 @@ class MessageActionsDialog(
     private val ncApi: NcApi
 ) : BottomSheetDialog(chatController.activity!!) {
 
+    @Inject
+    lateinit var viewThemeUtils: ViewThemeUtils
+
     private lateinit var dialogMessageActionsBinding: DialogMessageActionsBinding
 
     private lateinit var popup: EmojiPopup
 
+    init {
+        NextcloudTalkApplication.sharedApplication?.componentApplication?.inject(this)
+    }
```

Comment:
Why on `init` and not on `onCreate`? Genuine question, I don't know the benefits of either. The only thing I can think of is that, if done on `onCreate`, injection will be skipped if the dialog is never created after instantiation.

Reply:
No idea, I c&p it from some of the existing dialogs, but yeah, I'll move it to `onCreate`